NorsokSCDLibrary.aml

```diff
@@ -1240,82 +1245,79 @@
         <Value />
       </Attribute>
       <Attribute Name="NoteRef" AttributeDataType="xs:string">
-        <Description>Reference to Drawing Note</Description>
+        <Description>Reference to Drawing Note object GUID</Description>
         <DefaultValue />
         <Value />
+        <Constraint Name="aml-RegExp">
+          <UnknownType>
+            <Requirements>^([0-9A-Fa-f]{8}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{12})$</Requirements>
```

Comment:
The RegEx can be shortened/optimized to this:
```suggestion
            <Requirements>^[0-9A-Fa-f]{8}(-[0-9A-Fa-f]{4}){4}[0-9A-Fa-f]{8}$</Requirements>
```
or, if you prefer it slightly more intuitive (not splitting the last 12 into 4 + 8): `<Requirements>^[0-9A-Fa-f]{8}(-[0-9A-Fa-f]{4}){3}-[0-9A-Fa-f]{12}$</Requirements>`

Reply:
Changed in c586f11d3b056df9d2a8b5dba3b5c983158a2ded